src/commands/shell.js

```diff
@@ -0,0 +1,38 @@
+const { requireHerbarium } = require('../generators/utils')
+const repl = require('@herbsjs/herbs2repl')
+
+const cmd = {
+  name: 'shell',
+  description: 'Run Herbs Shell',
```

Comment:
@danielsous What do you think about "Run herbs shell to interact with your project easily"

Reply:
Good, I can change the description.